packages/orchestrator-ui-components/src/components/WfoTable/WfoBasicTable/WfoStatusColorField.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { EuiFlexItem } from '@elastic/eui';
+import { useOrchestratorTheme } from '../../../hooks';
+import { defaultOrchestratorTheme } from '../../../theme';
+import { WFO_STATUS_COLOR_FIELD } from '../utils/columns';
+
+export type WfoStatusColorFieldProps = {
+    colorMappings: { [key: string]: string[] };
+    state: string;
+};
+
+/*
+    Can we have all the definitions (ServiceTicketDefinition, ProductBlockDefinition ...)
+    extend from a base type? (ex. Definition)
+*/
+export const STATUS_COLOR_FIELD_COLUMN_PROPS = {
+    field: WFO_STATUS_COLOR_FIELD,
+    name: '',
+    width: '1',
+};
+
+type ThemeType = typeof defaultOrchestratorTheme;
+
+export const WfoStatusColorField = ({
```

Comment:
This component should only take care of coloring a rectangle. Basicly the only prop should be `color`. The logic how to determine this color based on state can be placed near the service ticket table.

Reply:
I tried a different approach here which I also wanted to apply later for the status badges as well so we can avoid duplication. Maybe we can discuss that in a call.